staging/src/k8s.io/apiserver/pkg/storage/etcd3/store_test.go

```diff
@@ -639,6 +639,108 @@ func TestInvalidKeys(t *testing.T) {
 	expectInvalidKey("Count", countErr)
 }
 
+func TestResolveGetListRev(t *testing.T) {
+	_, store, _ := testSetup(t)
+	testCases := []struct {
+		name          string
+		continueKey   string
+		continueRV    int64
+		rv            string
+		rvMatch       metav1.ResourceVersionMatch
+		recursive     bool
+		expectedError string
+		limit         int64
+		expectedRev   int64
+	}{
+		{
+			name:          "specifying resource versionwhen using continue",
+			continueKey:   "continue",
+			continueRV:    100,
+			rv:            "200",
+			expectedError: "specifying resource version is not allowed when using continue",
+		},
+		{
+			name:          "invalid resource version",
+			rv:            "invalid",
+			expectedError: "invalid resource version",
+		},
+		{
+			name:          "unknown ResourceVersionMatch value",
+			rv:            "200",
+			rvMatch:       "unknown",
+			expectedError: "unknown ResourceVersionMatch value",
+		},
+		{
+			name:        "use continueRV",
```

Comment:
Add another testcase which is identical to this one but has RV=""

Reply:
done